deploy/utils/backup_script/run_backup.sh

```diff
@@ -34,28 +38,30 @@ if [ -n "$response" ]; then
   echo "$response" | jq
 else
   echo "Error. No response was received from node"
+  rm -f $LOCKFILE
   exit 1
 fi
 
-export PATH=$PATH:$gcloud_path
-
-#Checking the path to gcloud
-if ! command -v gcloud &> /dev/null; then
+# Checking the path to gcloud
+if ! command -v $gcloud_bin &> /dev/null; then
   echo "Error. gcloud not found in PATH"
+  rm -f $LOCKFILE
   exit 1
 fi
 
 # Checking the gcp_key_file by the specified argument in CMD
 if [ ! -f "$gcp_key_file" ]; then
   echo "Error. GCP key file $gcp_key_file not found."
+  rm -f $LOCKFILE
   exit 1
 fi
 
-gcloud auth activate-service-account --key-file="$gcp_key_file"
+$gcloud_bin auth activate-service-account --key-file="$gcp_key_file"
 
-#Verification of authorization in GCP
+# Verification of authorization in GCP
 if [ $? -ne 0 ]; then
   echo "Error. GCP service account could not be activated."
+  rm -f $LOCKFILE
```

Comment:
This post-work probably should be done in a `terminate` function that handles like `trap 'terminate' SIGINT`.